post_installation_configuration/cluster-capabilities.adoc

```diff
@@ -6,26 +6,50 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Cluster administrators can use the cluster capabilities to select or deselect one or more optional components prior to installation and also enable a cluster capability anytime post installation.
+Cluster administrators can use cluster capabilities to disable one or more optional components prior to installation. Cluster administrators can enable cluster capabilities at anytime after installation.
 
 [NOTE]
 ====
-If cluster capability has been enabled before, it cannot be disabled.
+Cluster administrators cannot disable a cluster capability after it is enabled.
 ====
 
-include::modules/viewing-cluster-capabilities.adoc[leveloffset=+1]
+include::modules/explanation-of-capabilities.adoc[leveloffset=+1]
 
-include::modules/enabling-baseline-capability-set.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+* xref:../operators/operator-reference.adoc#cluster-operator-reference[Cluster Operators reference]
+
+include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install[Deploying installer-provisioned clusters on bare metal]
```

Comment:
If the purpose of the additional resources is letting users know how the baremetal cluster is installed, it would be nice to get the [upi install doc](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html) linked here as well.

Reply:
Since the BMO should not be disabled on IPI deployments, I changed the link to the UPI install doc and added a link to the Post-install bare metal configuration.